README.md

```diff
@@ -46,6 +46,39 @@ You can refer our [Code of Conduct](https://github.com/dell/dellemc-openmanage-a
 ## Testing
 See [here](https://github.com/dell/dellemc-openmanage-ansible-modules/tree/collections/tests/README.md) for further information on testing.
 
+## Debugging
+
+1. First you will need to download the code as you would normally with `ansible-galaxy collection install dellemc.openmanage`. This will pull the code down. On Ubuntu this defaults to `/home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/` The problem with this is that this location is not in PYTHONPATH which will cause your debugger problems.
+2. To resolve the pathing issues I moved the openmanage collection to align with the rest of Dell's code which is in PYTHONPATH with `sudo mv /home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/ /usr/local/lib/python3.8/dist-packages/ansible_collections/dellemc/`
+    1. The location may be different for you but the main thing is `openmanage` must be accessible within the `ansible_collections.dellemc` namespace. That is to say, your path should look like `<SOME_PREFIX (usually dist-packages)>/ansible_collections/dellemc/openmanage`
+3. You will know it worked if things like `from ansible_collections.dellemc.openmanage.plugins.module_utils.dellemc_idrac import iDRACConnection` import cleanly and without error. Some IDEs may require you to restart the IDE in order to rescan the available packages.
+4. **NOTE:** In my IDE, `from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError` still reported an error. This error can be ignored safely. The code should still run correctly. Ansible is doing some import magic to make this work that may confuse your IDE's ability to check for the import correctly but it does actually run.
+5. One thing that isn't immediately obvious about debugging Ansible modules is how you inject arguments. This is described [in the Ansible docs](https://docs.ansible.com/ansible/2.9/dev_guide/developing_modules_general.html#exercising-module-code-locally).
```

Comment:
It would be good to point to the latest URL https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html#exercising-module-code-locally

Reply:
@anupamaloke updated!